Xamarin.Forms.Platform.iOS/Renderers/ImageElementManager.cs

```diff
@@ -154,5 +132,23 @@ public static async Task SetImage(IImageVisualElementRenderer renderer, IImageEl
 
 			(imageElement as IViewController)?.NativeSizeChanged();
 		}
+
+		internal static async Task<UIImage> GetNativeImageAsync(this ImageSource source, CancellationToken cancellationToken = default(CancellationToken))
+		{
+			IImageSourceHandler handler;
+			if (source != null && (handler = Internals.Registrar.Registered.GetHandlerForObject<IImageSourceHandler>(source)) != null)
+			{
+				try
+				{
+					return await handler.LoadImageAsync(source, scale: (float)UIScreen.MainScreen.Scale, cancelationToken: cancellationToken);
+				}
+				catch (OperationCanceledException)
+				{
+					// no-op
```

Comment:
Might it make sense to log this cancellation? So users have some idea _why_ they're getting null images back.

Reply:
added